docs/advanced/subclassed-layers.md

```diff
@@ -7,14 +7,13 @@ the vertex and/or fragment shaders.
 If a small feature is missing from a layer, subclassing can often be a
 good technique to add it.
 
-## Overriding Attribute Calculation
+### Overriding Attribute Calculation
```

Comment:
About the hashes... normally we would "indent one level at a time. Assuming you decided that this looks better? 

@Pessimistress is there a way in our markdown component to control the size of various headings rather than using non-standard "indentation"?

I have concerns with this - we have more and more maintainers of this library and it is not easy for other engineers to figure out that we are using non-standard indentation.